ftrip.io.catalog-service/ftrip.io.catalog-service/Properties/launchSettings.json

```diff
@@ -21,7 +21,7 @@
       "commandName": "Project",
       "launchBrowser": true,
       "launchUrl": "swagger/index.html",
-      "applicationUrl": "http://localhost:4999",
+      "applicationUrl": "http://localhost:4997",
```

Comment:
Can you change this to 4994. This port is taken by EmailService and we cannot run both of them locally.